pkg/render/terminal.go

```diff
@@ -319,6 +321,13 @@ func renderFileSummary(_ context.Context, fr *malcontent.FileReport, w io.Writer
 					content = fmt.Sprintf("%s%s%s %s %s", prefix, indent, bullet, rest, desc)
 					e = ""
 				}
+
+				if b.NoDiff {
+					prefix = "~~~"
```

Comment:
To increase familiarity with folks who review diffs from diff or GitHub, can we leave NoDiff lines without a prefix or special appearance? I like the disambiguation that ~~~ gives, but since it isn't used elsewhere, I think it will raise more questions than answers.

Reply:
Updated in [`4d0921c` (#628)](https://github.com/chainguard-dev/malcontent/pull/628/commits/4d0921c86b60e9e77053d4104a4df662f14a7e1f).